src/arc_utilities/xbox.py

```diff
@@ -40,4 +49,34 @@ def wait_for_button(self, button, message=True):
         if message:
             rospy.loginfo("Waiting for xbox button: " + button)
 
-        wait_for(lambda: not self.get_button(button) == 0)
\ No newline at end of file
+        wait_for(lambda: not self.get_button(button) == 0)
+
+    def x_clicked(self, event: InputEvent):
```

Comment:
What's the cost/benefit of using `x_clicked` vs `get_button`? Just from the interface, I am unclear when I should use `x_clicked`. Relatedly, where to these `events` come from?

`x_clicked` can be static

Reply:
yea it's super hacky, the event loop that calls this is in the main cartesian script, whereas get_button uses the background thread + ros topic thing. Merging for now, if someone else wants to use it we can clean it up later